.config

```diff
@@ -147,3 +147,6 @@ CT_LEGENDS_VERSION="1.19.0"
 CT_ANNOUNCEMENT=y
 CT_ANNOUNCEMENT_VERSION="v1.3.0"
 # CT_APPIMAGE is not set
+CT_DARK=n
+CT_DA_VERSION="4_47_04_v39"
```

Comment:
shouldn't it be CT_DARK, CT_DARK_VERSION ?

Reply:
Good point!